web/src/app/dialogs/FormDialog.js

```diff
@@ -193,7 +197,7 @@ export default class FormDialog extends React.PureComponent {
     }
 
     return (
-      <DialogActions>
```

Comment:
i think we can leave this as it was in the previous commit, as we still want the header to remain sticky at the top as well